src/module-1/index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles/css/style.css">
+  <link rel="shortcut icon" type="image/png" href="img/favicon.ico">
+
+  <title>Online Store</title>
+</head>
+<body>
+<div class="wrapper">
+  <div class="product">
+    <img class="product__img" src="img/products/apple-watch@2x.png" />
+    <div class="flex-row">
+      <div class="product__rate">
+        <button class="product__rate-btn">
+          <span class="product__rate-num">4.4</span>
+          <span class="product__rate-star"></span>
+        </button>
+      </div>
+      <div class="product__price">$399</div>
+    </div>
+    <p class="product__title">Apple Watch Series 4 GPS</p>
+    <p class="product__description">
+      Redesigned from scratch and completely revised.
+    </p>
+    <div class="flex-row">
+      <div class="product__btn">
+        <button class="product__btn-group-1">
+          <span class="product__btn--heart"></span>
+          <span class="product__btn-text-1">WISHLIST</span>
+        </button>
+      </div>
+      <div class="product__btn">
+        <button class="product__btn-group-2">
+          <span class="product__btn--shopping-bag"></span>
+          <span class="product__btn-text-2">ADD TO CART</span>
+        </button>
+      </div>
+    </div>
+  </div>
+
+  <div class="product">
+    <img class="product__img" src="img/products/jbl-speaker@2x.png" />
+    <div class="product__price">$199</div>
+    <p class="product__title">JBL Speaker</p>
+    <p class="product__description">
+      Redesigned from scratch and completely revised.
+    </p>
+    <div class="flex-row">
+      <div class="product__btn">
+        <button class="product__btn-group-1">
+          <span class="product__btn--heart"></span>
```

Comment:
```suggestion
          <span class="product__btn-heart"></span>
```  
I'd not add `--` here because this is not a style for some state, like --active, --disabled, --selected etc. The same for cart.